lib/SPIRV/OCLUtil.cpp

```diff
@@ -647,6 +647,7 @@ template <> void LLVMSPIRVAtomicRmwOpCodeMap::init() {
   add(llvm::AtomicRMWInst::UMax, OpAtomicUMax);
   add(llvm::AtomicRMWInst::UMin, OpAtomicUMin);
   add(llvm::AtomicRMWInst::FAdd, OpAtomicFAddEXT);
+  add(llvm::AtomicRMWInst::FSub, OpAtomicFAddEXT);
```

Comment:
I beleive we shouln't add such entry to the map making fsub being implicitly replaced by fadd (it can probably lead to bugs in the future) - instead FNegative and AtomicFAddEXT should be both generated explicitly.

Reply:
Thanks for the review, good point, I'll special case fsub instead of relying on the existing path.

Reply:
Please check the latest commit when you get a chance, I think I addressed this feedback while reusing existing code but maybe I misunderstood